docs/index.md

```diff
@@ -94,6 +94,7 @@ Some language servers support multiple languages, which can be specified in the
 | tcp_port | see instructions below |
 | tcp_host | see instructions below |
 | tcp_mode | see instructions below |
+| experimental_capabilities | options to send to the server at startup (rarely used) |
```

Comment:
This now has the exact same description as `initializationOptions`. This might confuse new users. Is there a better description for experimental_capabilities? Perhaps simply "Turn on experimental capabilities of a language server. This is a dictionary and differs per language server".